src/init/starship.bash

```diff
@@ -52,21 +52,30 @@ starship_precmd() {
 
     eval "$_PRESERVED_PROMPT_COMMAND"
 
+    local ARGS="--terminal-width=${COLUMNS} --status=${STARSHIP_CMD_STATUS} --pipestatus=${STARSHIP_PIPE_STATUS[*]} --jobs=${NUM_JOBS}"
```

Comment:
```suggestion
    local -a ARGS=(--terminal-width="${COLUMNS}" --status="${STARSHIP_CMD_STATUS}" --pipestatus="${STARSHIP_PIPE_STATUS[*]}" --jobs="${NUM_JOBS}")
```

I'd suggest turning `ARGS` into an array. The later `${ARGS}` should be replaced by `"${ARGS[@]}"`. Also, `ARGS+=...` should be `ARGS+=(...)` with proper quoting.

Reply:
Hey, thanks for the suggestions - I have added them.